frontend/admin-client/src/app/shared/components/pagination/pagination.component.html

```diff
@@ -1,27 +1,31 @@
 <nav>
   <ul class="pagination">
-    <li class="page-item" [ngClass]="(page === 1) ? 'disabled': ''">
+    <li class="page-item" [ngClass]="{ 'disabled' : currentPage === 1 }">
       <i class="page-link fas fa-angle-double-left" (click)="goFirstPage()"></i>
     </li>
-    <li class="page-item" [ngClass]="(page === 1) ? 'disabled': ''">
+    <li class="page-item" [ngClass]="{ 'disabled' : currentPage === 1 }">
       <i class="fas fa-angle-left page-link" (click)="goPrevPage()"></i>
     </li>
-    <li class="page-item disabled" *ngIf="activePages[0] != 1">
-      <i class="page-link fas fa-ellipsis-h"></i>
-    </li>
-    <li *ngFor="let p of activePages;" class="page-item" [ngClass]="(p === page) ? 'active': ''">
-      <a class="page-link" (click)="goToPage(p)">
-        {{p}}
+    <ng-container *ngIf="currentPage > visiblePageCount + 1">
```

Comment:
Why do you need ng-container here?

Reply:
I though it made it more readable to wrap it in an ng-if. That is it. It has no impact on the html.